vllm_ascend/worker/model_runner_v1.py

```diff
@@ -228,7 +232,7 @@ def __init__(self, vllm_config: VllmConfig, device: torch.device):
         self.requests: Dict[str, CachedRequestState] = {}
         # Persistent batch.
         # Remove this after we drop 0.8.5 support
-        if vllm_version_is("0.8.5") or vllm_version_is("0.8.5.post1"):
+        if vllm_version_is("0.8.5") or ("0.8.5.post1"):
```

Comment:
this if statement will always be True, change this back to the last version
```suggestion
        if vllm_version_is("0.8.5") or vllm_version_is("0.8.5.post1"):
```